### PR TITLE
Added jpg support alongside with sjpg

### PR DESCRIPTION
**Pull request by vinodstanur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,61 @@
...
+   lv_obj_t * img2;
 
+   lv_split_jpeg_init();
+   img1 = lv_img_create(lv_scr_act(), NULL);
+   img2 = lv_img_create(lv_scr_act(), NULL);
+   
+   lv_img_set_src(img1,  &wallpaper_jpg);         //jpg image 1 from c array inside sample_sjpg_images
+   //lv_img_set_src(img1,  &wallpaper);            //sjpg image 1 from c array inside sample_sjpg_images
+   
+   lv_img_set_src(img2,  "S.\\lv_examples\\assets\\lv_lib_split_jpg\\sample_sjpg_images\\small_image.sjpg"); //sjpg file from file system ( using lv_fs )
```
Comment (kisvegabor) [inline README.md]: It'd be better to store the examples in this repo.

File: lv_sjpg.h [hunk 2]
```diff
@@ -1,65 +1,51 @@
...
 
+/*********************
+ *      DEFINES
+ *********************/
+//#define LV_SJPG_FREE(x)                 free(x)
+//#define LV_SJPG_MALLOC(x)               malloc(x)
+#define TJPGD_WORKBUFF_SIZE             4096    //Recommended by TJPGD libray
+
+//NEVER EDIT THESE OFFSET VALUES
 #define SJPEG_VERSION_OFFSET            8
```
Comment (kisvegabor) [inline lv_sjpg.h]: Are these required in the header file? If not could you move it to the C file?

**Review by kisvegabor — COMMENTED** [review]
Thanks for the update. Looks great in general, I have only some minor comments. 

I can test it tomorrow.

File: lv_sjpg.h (hunk 2 above)
Comment (vinodstanur) [reply lv_sjpg.h]: Yes will move to the c.

File: README.md (hunk 1 above)
Comment (vinodstanur) [reply README.md]: This is done. Added this repo alongside with lvgl and added an example folder inside this repo with a very basic demo code using the images inside the example/images.

**Comment by kisvegabor** [comment]
I've tested it and updated a few stuff in the README.

I found 2 "real"  issue:
1. It seems only 16-bit color format is supported. Would it be possible to handle other color depths too? 
2. I run the script from the `script` folder like this:
```
python3 jpg_to_sjpg.py ../example/images/wallpaper.jpg 

Conversion started...

Input:
	../example/images/wallpaper.jpg
	RES = 640 x 480

Output:
	Time taken = 0.89 sec
	bin size = 84.2 KB
	.sjpg	(bin file)
	.c		(c array)

All good!
```

As you can see it generated to empty-named files. If the image is in the same folder as the script it works well.

**Comment by vinodstanur** [comment]
Hi,
  Yes, as of now it is only 16 bit support and I kept it as to-do, we can do it soon.

I will check the python script for the second issue.

**Comment by kisvegabor** [comment]
> Yes, as of now it is only 16 bit support and I kept it as to-do, we can do it soon.

Alright, no problem, but this limitation should be noted in the README.

> I will check the python script for the second issue.

Thank you!

**Comment by kisvegabor** [comment]
Thanks! I'll be away from PC this weekend so I can test it only on Monday.

**Comment by kisvegabor** [comment]
I've tested it and worked well here. 
Thank you very much for the updates.

**Comment by vinodstanur** [comment]
Thanks.

I realised I haven't updated the readme overview.

**Comment by kisvegabor** [comment]
As this repo is completely your work, I'd happily give you write access here. It'd make it easier to add fixes later and (I think) it feels to have your repo in an origination. :slightly_smiling_face:

**Pull request merged**